packages/trpc/server/createContext.ts

```diff
@@ -87,7 +87,8 @@ async function getUserFromSession({
     return null;
   }
   // This helps to prevent reaching the 4MB payload limit by avoiding base64 and instead passing the avatar url
-  if (user.avatar) user.avatar = `${CAL_URL}/${user.username}/avatar.png`;
```

Comment:
This is the big @harshsinghatz

Reply:
@hariombalhara does this change returns the whole base64 avatar in the response? If it does we need to address this ASAP.

Reply:
Only viewer.me endpoint is having the size issue now. Opened a PR to fix that.
https://github.com/calcom/cal.com/pull/5392